### PR TITLE
use the default tx level

### PR DESCRIPTION
**Pull request by stalep**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: horreum-backend/src/main/resources/application.properties [hunk 1]
```diff
@@ -11,7 +11,6 @@ quarkus.datasource.jdbc.max-size=10
 # With the default isolation level (read committed) we could experience a situation where
```
[the remark this answers is not in the document]
Comment (stalep) [reply horreum-backend/src/main/resources/application.properties:11]: No, probably not.

**Comment by stalep** [comment]
I see that the CI is failing, I can't reproduce that locally though...

**Comment by jesperpedersen** [comment]
It is working here as well - will need to check more

**Comment by jesperpedersen** [comment]
Going to close for now, as there are issues with transaction propagation in general

**Pull request closed without merging**